snuba/cli/bootstrap.py

```diff
@@ -40,13 +40,15 @@ def bootstrap(bootstrap_server, kafka, force):
         topics = []
         for name in DATASET_NAMES:
             dataset = get_dataset(name)
-            partitions = dataset.get_default_partitions()
-            replication = dataset.get_default_replication_factor()
-            topics.extend([
-                (dataset.get_default_topic(), partitions, replication),
-                (dataset.get_default_replacement_topic(), partitions, replication),
-                (dataset.get_default_commit_log_topic(), partitions, replication),
-            ])
+            if dataset.can_write():
+                stream_loader = dataset.get_table_writer().get_stream_loader()
+                partitions = stream_loader.get_default_partitions()
+                replication = stream_loader.get_default_replication_factor()
+                topics.extend([
+                    (stream_loader.get_default_topic(), partitions, replication),
+                    (dataset.get_default_replacement_topic(), partitions, replication),
+                    (dataset.get_default_commit_log_topic(), partitions, replication),
+                ])
```

Comment:
ok, but I still need to have methods for each topic in StreamLoader, because, except for this case, all the others require a specific topic, not the list.